### ISSUE TITLE
Domains: Nameservers toggle lacking a label

### ISSUE THREAD

**Issue by tyxla** [Automattic/wp-calypso#53003]
#### Steps to reproduce

1. Go to `/domains/manage/:domain/name-servers/:site`
2. You'll notice that the "Use WordPress.com Name Servers" toggle is missing a label.

#### What I expected

We should add a label so that the toggle would be more accessible.

#### What happened instead

The label is not technically a `label`, rather it's a `span`.

#### Context / Source

Reported by @sarayourfriend in https://github.com/Automattic/wp-calypso/pull/52954#discussion_r634375852 while testing and reviewing #52954.

**Comment by simison** [comment]
cc @kelliepeterson good one for team Nomado?

**Comment by kelliepeterson** [comment]
Thanks @simison yep, we can grab this but I actually want to do a more substantive change to this.

**Comment by Christopher-Stevers** [comment]
@kelliepeterson just wondering how you are doing on this issue. I'd like to pick it up as my first issue.

**Comment by spraveenitpro** [comment]
Hi @tyxla I would like  to work on this.

**Comment by tyxla** [comment]
Awesome, let me know if you have any questions @spraveenitpro! cc @Automattic/nomado

**Comment by tyxla** [comment]
This one was fixed by @spraveenitpro in #57374, thanks a bunch for the contribution! 🙌
